packages/views/screens/managers/wallet/explorerLayout.js

```diff
@@ -0,0 +1,68 @@
+/* istanbul ignore file */
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { isEmpty } from '@common/utilities/helpers';
+import { selectActiveToken, selectSettings } from '@common/store/selectors';
+import { selectSearchParamValue } from '@screens/router/searchParams';
+import TabsContainer from '@basics/tabs/tabsContainer/tabsContainer';
+import Overview from './overview';
+import DelegateTab from './delegateProfile';
+import VotesTab from './votes';
+import Transactions from './transactions';
+
+const ExplorerLayout = ({
+  t, account, history,
+}) => {
+  const activeToken = useSelector(selectActiveToken);
+  const { discreetMode } = useSelector(selectSettings);
+  const address = selectSearchParamValue(history.location.search, 'address');
+
+  useEffect(() => {
+    account.loadData();
+  }, [address]);
```

Comment:
```suggestion
  useEffect(account.loadData, [address]);
```

I think since `account.loadData` isn't async or doesn't return anything I think it would be bet to just have 1 function call instead of two

Reply:
@eniolam1000752 To the best of my knowledge, `account.loadData` is async as it loads the account details and passes it down from the wrapper/manager component